roc/roc-wrapper.go

```diff
@@ -38,48 +49,94 @@ func OpenContext(config *ContextConfig) *Context {
 }
 
 func (c *Context) Close() error {
-	return convertErr(contextClose(c), "Error when closing context")
+	errCode := contextClose(c)
+	if errCode == 0 {
+		return nil
+	}
+	if errCode < 0 {
+		return errors.New("Arguments are invalid or or there are objects attached to the context")
+	}
+	return ErrInvalidApi
```

Comment:
Nitpick: can we use the same layout in all wrappers? Either:

```
	if errCode == 0 {
		...
	}
	if errCode < 0 {
		...
	}
	return ...
```

or:

```
	if errCode == 0 {
		...
	}
	if errCode > 0 {
		...
	}
	return ...
```

Reply:
done